### DISCUSSION TITLE
Regarding chromatic interactions plots

### DISCUSSION THREAD

**Discussion by jeromejh** [GreenleafLab/ArchR#1838] [Questions / Documentation]
Good morning, 

I've been trying to understand how to get plots in the tutorial 15.2 Co-accessibility with ArchR.
(https://www.archrproject.com/bookdown/co-accessibility-with-archr.html)

It seems that projHeme5 needs to be created from projHeme1 -> projHeme2 -> projHeme3 -> projHeme4, and I'm a bit lost to start from projHeme1. Are there any ways that I could simply input the data I have and draw plots? 

Thank you.

Best regards,
Jerome

**Comment by rcorces** [comment]
Its unclear what you mean by

> simply input the data I have and draw plots

Co-accessibility has certain pre-reqs that are established at earlier time points in the manual, for example peak calls.
